src/background/index.js

```diff
@@ -248,66 +237,5 @@ initialize()
   sync.initialize();
   resetBlacklist();
   autoCheckRemove();
+  global.dispatchEvent(new Event('backgroundInitialized'));
```

Comment:
How about calling the function directly?

Reply:
I didn't want to make the parent context do the child's work explicitly. Event-driven approach would allow multiple children to subscribe. WDYT?

Reply:
Sounds convincing.